features/invoices_has_to_be_paid.feature

```diff
@@ -0,0 +1,27 @@
+Feature: Invoices has to be paid to be viewed
+  As a company
+  In order to get paid 
+  I must restrict access to invoices
+
+  Background:
+    Given the following user exist
+      | email         | password |
+      | user@user.com | password |
+
+    And I am logged in as "user@user.com"
```

Comment:
this should be underneath the `Given the following invoices exist`

Reply:
Test need that order to create invoices

Reply:
not anymore